uktrade/data-hub-frontend
# Reactify OMIS edit contact form

## Description of change
The form for editing the contact for an OMIS order has been reactified. As part of this work I've incorporated the add contact loop into this page, which wasn't in use previously.

As this is the final edit form to be reactified, the OMIS edit controller and associated code has also been deleted.

## Test instructions

Go to an OMIS order and edit the contact. Everything should work as before (note that the status message shown in the screenshot only appears where the order status is not `draft`).

To test the contact loop, open the details component and click the link. You should be redirected to the OMIS contact page once you've addded the new contact.

## Screenshots

### Before

<img width="756" alt="Screenshot 2023-11-01 at 17 25 12" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/d006228b-15a6-468c-be33-1187682e84eb">


### After

<img width="672" alt="Screenshot 2023-11-01 at 17 25 02" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/6f7e4b01-c101-4e7b-9cdc-0042ecba60c8">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
